test/e2e/specs/end_user.go

```diff
@@ -21,7 +21,7 @@ var _ = Describe("Openshift on Azure end user e2e tests [EndUser][Fake]", func()
 		Expect(cli).ToNot(BeNil())
 	})
 
-	It("should create and validate test apps", func() {
+	It("should create and validate test apps [EndUser][Fake][Apps]", func() {
```

Comment:
Whats the purpose of the tags added to the `It` blocks in this file?

Reply:
@charlesakalugwu i missed your comment here: What it does is allow you to run a single spec straight while maintaining backwards compat, example:

* you want to run all the validation suites, you run `FOCUS="\[EndUser\]\[Fake\]" ./hack/e2e.sh` same as before
* you want to *only* run the cluster validation suite, you run `FOCUS="\[EndUser\]\[Fake\]\[Cluster\]" ./hack/e2e.sh`
* you want to *only* run the app validation suite, you run `FOCUS="\[EndUser\]\[Fake\]\[Apps\]" ./hack/e2e.sh`